bindings/core/src/method_handler/utils.rs

```diff
@@ -27,9 +27,7 @@ pub(crate) fn call_utils_method_internal(method: UtilsMethod) -> Result<Response
         UtilsMethod::HexPublicKeyToBech32Address { hex, bech32_hrp } => {
             Response::Bech32Address(hex_public_key_to_bech32_address(&hex, &bech32_hrp)?)
         }
-        UtilsMethod::ParseBech32Address { address } => {
-            Response::ParsedBech32Address(AddressDto::from(&Address::try_from_bech32(address)?))
-        }
+        UtilsMethod::ParseBech32Address { address } => Response::ParsedBech32Address(AddressDto::from(address.inner())),
```

Comment:
Wondering why that `ParsedBech32Address` variant even exists, maybe we should just return Bech32 response?

Reply:
Removed it.

Reply:
ParsedBech32Address contains the type and not the bech32 string

Reply:
Sure but idn't it better to have Bech32? It also contains the but but doesn't discard the HRP

Reply:
The function is there to return the address dto if you provide it as bech32 :carlos:

Reply:
I can revert it